src/OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -121,6 +122,9 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
     absl::StrFormat(format, ##__VA_ARGS__)                                                     \
   }
 
+#define ORBIT_LOG_VAR(x) ORBIT_LOG("%s = %s", #x, orbit_base::ToString(x))
+#define ORBIT_LOG_VARN(x, name) ORBIT_LOG("%s = %s", name, orbit_base::ToString(x))
```

Comment:
Is the `VARN` version valuable? You have to write the name twice anyway. Sure, you might save on `%xyz` and you might save on the conversion of wide strings, but these seem marginal gains or specific cases.

Reply:
I think so, yes, as it still saves the hassle of converting to a string manually. This is more evident if we have specializations of `ToString` for user-defined types which would be more complex than a simple `%xyz`, but it's also handy in general because we don't have to do the work of figuring out `%xyz`.

Reply:
> This is more evident if we have specializations of ToString for user-defined types

How would that work? Would you have a version of `orbit_base::ToString` that tries to pick up some `ToString` defined by the type? Isn't it too much infrastructure around these macros?

> we don't have to do the work of figuring out %xyz

Considering that the `VARN` version would be used like 1% of the times, is that measurable? With `absl::StrFormat` in `ORBIT_LOG` it's %s or %d or %f most of the times. Other times it's %x and `ORBIT_LOG_VARN` can't know you want that.